experimental/class-features.md

```diff
@@ -0,0 +1,63 @@
+
+# Class Features
+
+This experimental extension covers three class features proposals: 
+[Class Fields], [Static Class Features] and [Private Methods].
+
+## ClassBody
+
+```js
+extend interface ClassBody <: Node {
+    body: [ MethodDefinition | PropertyDefinition ];
+}
+```
+
+## PropertyDefinition
+
+```js
+interface PropertyDefinition <: Node {
+    type: "PropertyDefinition";
+    key: Expression | PrivateName;
+    value: Expression;
+    computed: boolean;
+    static: boolean;
+    private: boolean;
```

Comment:
I love consistency. If we add `private:boolean` to `(Method|Property)Definition`, I'd like to add it to `MemberExpression` as well. Or remove it from both.

Reply:
I think it's best to remove from both; otherwise we're duplicating same information which explicitly goes against one of the design goeals ("Unique") in the Philosophy.

Reply:
I agree on @RReverser that we don't need `.private` if we can check `.key.type === "PrivateIdentifier"`. While I admit that `.private` is handier, the ergonomic trade-off here is still acceptable.

Should another private-like key (`PrivateComputedExpression`?) be supported in the future, we can introduce `.private` at that time.